### ISSUE TITLE
Dokumentenverwaltung

### ISSUE THREAD

**Issue by ThomasBartelworth** [SVWS-NRW/Schild3-BetaTest#1032]
Bei der Einrichtung der Dokumentenverwaltung wird für Kinder in der Neuaufnahme (Häkchen gesetzt) kein Ordner erstellt.

**Comment by JuergenRichter** [comment]
Das funktioniert nur, wenn bei den betreffenden Kindern auch schon eine Klasse hinterlegt ist (die Maske für die Verzeichnisstruktur kann ja die Klasse umfassen). War das der Fall?

**Comment by AnneSchueller** [comment]
Funktioniert für Schüler aus der Neuaufnahme, bei denen eine Klasse bereits zugeordnet ist.

**Comment by ThomasBartelworth** [comment]
Ja, es war der Wunsch (einer Schule) schon bei der Schulanmeldung die Dokumentenablage nutzen zu können, um dort Dokumente (Masernnachweis etc. schon vor der Klassenbildung ablegen zu können.
